projects/batfish/src/org/batfish/grammar/flatjuniper/FlatJuniperLexer.g4

```diff
@@ -731,6 +731,11 @@ ECHO_REQUEST
    'echo-request'
 ;
 
+EDIT
```

Comment:
I am not very comfortable with parsing Juniper 'edit' lines. Those are really commands, not config directives. I need to see config snippets justifying this.